### PR TITLE
Add preserveAspectRatioTag prop and strip aspectRatio by default.

### PR DESCRIPTION
**Pull request by eiwi1101**
An (extremely delayed) followup to https://github.com/Dean177/react-justified-layout/issues/3

### CODE REVIEW

File: src/react-justified-layout.js [hunk 1]
```diff
@@ -94,10 +96,17 @@ class JustifiedLayout extends React.Component {
       <div style={{ ...Style, ...style, height: containerHeight, width: this.props.containerWidth }}>
         {map(elementLayout, ([element, layout]) => {
           const { height, left, top, width } = layout;
+
+          // Remove aspectRatio tag when rendering to prevent React errors:
+          let elementProps = element.props;
+          if (!preserveAspectRatioTag) {
+            delete elementProps.aspectRatio;
```
Comment (Dean177) [inline src/react-justified-layout.js]: Isn’t this mutating the props object?

Could we de-structure to avoid this?

`{ aspectRatio, ...elementProps }`

Comment (eiwi1101) [reply src/react-justified-layout.js]: Oh hey, you're right. 
```suggestion
            { aspectRatio, ...elementProps };
```

I'm guessing the destructuring there will be smart enough to overwrite the `elementProps` in the outer scope? I've never thought of doing that before.

Comment (Dean177) [reply src/react-justified-layout.js]: It doesn't sadly (give it a go in a `node` REPL and see what happens).

```js
let { aspectRation, ...rest } = element.props;
elementProps = rest;
```

Once you have made this change, add yourself as a contributor to package.json and I will merge and do a new release.

Thanks!

**Comment by eiwi1101** [comment]
Alright, changes in https://github.com/Dean177/react-justified-layout/pull/5/commits/71fb4dfe40556f2f1e099f524d8cbaf7a395b589

**Pull request merged**